woocommerce/woocommerce-android
# [Better Tablet Products] Fix for moving to trash on tablet. Empty state for the case when product is not selected

Closes: #11014
Closes: #11013


### Description

The PR:
* Fixes the issue when trash a product doesn't work on tablets
* Adds "error" state when product details opened with product that is not available anymore (the same used for "empty" state so we can show that when nothing is selected on 2 pane mode, e.g. when there are no products)

### Testing instructions

Do both on a tablet and a phone
* Open products
* Trash a product -> notice that it works now on a tablet
* Either trash all products you have or use a store with no products
* Notice that a meaningful message is shown in this case now instead of indefinitely loading screen

### Images/gif



https://github.com/woocommerce/woocommerce-android/assets/4923871/24566c80-f1c3-4aa1-9a66-56a47c5f7661
